web/src/components/button/CardButton/index.tsx

```diff
@@ -286,6 +286,12 @@ function CardButtonSubmenu({ button }) {
             }}
             label={t('button.delete')}
           />
+          <CardSubmenuOption
+            onClick={() => {
+              router.push(`/ButtonPin/${button.id}`);
```

Comment:
in here u can make it more simple:
in top u make
const pinButton = (buttonId) => {
   store.emit(new PinButton(buttonId));
}


and then here on the html u make:
onClick={() => pinButton(button.id)